source/agent/addons/quic/QuicTransportConnection.cc

```diff
@@ -0,0 +1,122 @@
+{
+    ELOG_DEBUG("OnIncomingStream.");
+    {
+        std::lock_guard<std::mutex> lock(m_streamQueueMutex);
+        m_streamsToBeNotified.push(stream);
+    }
+    m_asyncOnStream.data = this;
+    uv_async_send(&m_asyncOnStream);
+}
+
+void QuicTransportConnection::onEnded()
+{
+}
+
+NAN_METHOD(QuicTransportConnection::newInstance)
+{
+    if (!info.IsConstructCall()) {
+        ELOG_DEBUG("Not construct call.");
+        return;
+    }
+    QuicTransportConnection* obj = new QuicTransportConnection();
+    obj->Wrap(info.This());
+    uv_async_init(uv_default_loop(), &obj->m_asyncOnStream, &QuicTransportConnection::onStreamCallback);
+    info.GetReturnValue().Set(info.This());
+}
+
+v8::Local<v8::Object> QuicTransportConnection::newInstance(owt::quic::QuicTransportSessionInterface* session)
+{
+    Local<Object> connectionObject = Nan::NewInstance(Nan::New(QuicTransportConnection::s_constructor)).ToLocalChecked();
+    QuicTransportConnection* obj = Nan::ObjectWrap::Unwrap<QuicTransportConnection>(connectionObject);
+    obj->m_session = session;
+    return connectionObject;
+}
+
+NAUV_WORK_CB(QuicTransportConnection::onStreamCallback)
+{
+    ELOG_DEBUG("OnStreamCallback.");
+    Nan::HandleScope scope;
+    QuicTransportConnection* obj = reinterpret_cast<QuicTransportConnection*>(async->data);
+    if (obj == nullptr || obj->m_streamsToBeNotified.empty()) {
+        return;
+    }
+    std::lock_guard<std::mutex> lock(obj->m_streamQueueMutex);
+    while (!obj->m_streamsToBeNotified.empty()) {
+        v8::Local<v8::Object> streamObject = QuicTransportStream::newInstance(obj->m_streamsToBeNotified.front());
+        QuicTransportStream* stream = Nan::ObjectWrap::Unwrap<QuicTransportStream>(streamObject);
+        obj->m_streamsToBeNotified.front()->SetVisitor(stream);
+        Nan::MaybeLocal<v8::Value> onEvent = Nan::Get(obj->handle(), Nan::New<v8::String>("onincomingstream").ToLocalChecked());
+        if (!onEvent.IsEmpty()) {
+            v8::Local<v8::Value> onEventLocal = onEvent.ToLocalChecked();
+            if (onEventLocal->IsFunction()) {
+                v8::Local<v8::Function> eventCallback = onEventLocal.As<Function>();
+                Nan::AsyncResource* resource = new Nan::AsyncResource(Nan::New<v8::String>("onincomingstream").ToLocalChecked());
+                Local<Value> args[] = { streamObject };
+                resource->runInAsyncScope(Nan::GetCurrentContext()->Global(), eventCallback, 1, args);
+            }
+        } else {
+            ELOG_DEBUG("onEvent is empty");
+        }
+        obj->m_streamsToBeNotified.pop();
+    }
+}
+
+NAN_METHOD(QuicTransportConnection::createBidirectionalStream){
+    QuicTransportConnection* obj = Nan::ObjectWrap::Unwrap<QuicTransportConnection>(info.Holder());
+    auto stream=obj->m_session->CreateBidirectionalStream();
+    v8::Local<v8::Object> streamObject = QuicTransportStream::newInstance(stream);
+    info.GetReturnValue().Set(streamObject);
+}
```

Comment:
no new line at end

Reply:
`.clang-format` file in source directory indicates the code here follows WebKit style. It doesn't require new line at the end of file.